scripts/image_upload_glance.py

```diff
@@ -55,7 +55,7 @@ def upload_to_glance(image, image_name_in_glance, provider, disk_format):
     glance.images.upload(glance_img.id, open(image, 'rb'))
 
 
-def main():
+def main(args):
```

Comment:
So I think I'd like to see this re-arranged to function a bit cleaner.

1. rename this method to `run`
2. Move argparse into separate method `parse_args` (something like that)
3. `if __name__ == '__main__'` block modified to call `parse_args` and pass it to `run`

I think doing it that way is cleaner, and allows for easy calling from within a module or on the command line.

Reply:
@mshriver , done